src/main/scala/jigg/pipeline/Pipeline.scala

```diff
@@ -202,22 +203,38 @@ Currently the annotators listed below are installed. See the detail of each anno
       annotate(reader, true)
     }
 
-    // The output of basic XML.save method is not formatted, so we instead use PrettyPrinter.
-    // However, this method have to convert an entire XML into a String object, which would be problematic for huge dataset.
-    // XML.save(in + ".xml", xml, "UTF-8")
-    val printer = new scala.xml.PrettyPrinter(500, 2)
+    // If option of json is true, the output of this method is formatted JSON.
+    prop("outputFormat") match {
```

Comment:
Due to the mechanism of `@Prop` (sorry this is not yet well documented), `outputFormat` variable is automatically overwritten with the value of `prop('outputFormat')` (the default value is `xml`). Thus, the pattern matching with Some is unnecessary:

```scala
outputFormat match {
  case "json" => ...
  case _ => ...
}
```